src/lib.rs

```diff
@@ -405,6 +405,11 @@ pub enum CheckResult {
     /// Spam
     Spam,
     /// Guaranteed spam (via pro-tip header)
+    ///
+    /// "If the X-akismet-pro-tip header is set to discard, then Akismet has determined that the
+    /// comment is blatant spam, and you can safely discard it without saving it in any spam
+    /// queue. Read more about this feature in [this Akismet blog
+    /// post](http://blog.akismet.com/2014/04/23/theres-a-ninja-in-your-akismet/)."
     Discard,
```

Comment:
The blog post doesn't expand on the flag, it's mostly about their WordPress feature.
```suggestion
    /// "If the X-akismet-pro-tip header is set to discard, then Akismet has determined that the
    /// comment is blatant spam, and you can safely discard it without saving it in any spam
    /// queue."
    Discard,
```